pkg/cloud/vsphere/provisioner/common/templates.go

```diff
@@ -406,8 +406,27 @@ KUBELET=$(getversion kubelet ${KUBELET_VERSION}-)
 KUBEADM=$(getversion kubeadm ${KUBELET_VERSION}-)
 KUBECTL=$(getversion kubectl ${KUBELET_VERSION}-)
 apt-get install -y docker.io=${DOCKER_VER}
-# Explicit cni version is a temporary workaround till the right version can be automatically detected correctly
-apt-get install -y kubelet=${KUBELET} kubeadm=${KUBEADM} kubectl=${KUBECTL}
+
+### TEMPORARY solution
```

Comment:
LGTM on the temporary side.
these solutions are not ideal, but we are waiting on SIG Release to decide on how to proceed with fixing the packages (e.g. revisions).
cc @tpepper

^ you can also ask Tim on slack.